addons/survey/views/survey_templates_print.xml

```diff
@@ -58,7 +58,7 @@
                                                 </t>
                                                 <t t-else="">
                                                     <t t-if="question.question_type == 'text_box'" t-call="survey.question_text_box"/>
-                                                    <t t-if="question.question_type == 'char_box'" t-call="survey.question_char_box"/>
+                                                    <t t-if="question.question_type == 'char_box'" t-call="survey.question_char_box_multiline"/>
```

Comment:
Could we instead do:

```xml
<div t-if="question.question_type == 'char_box'" class="o_survey_print o_survey_comment_container p-0">
    <span t-out="answer_lines[0].value_char_box or None"/>
</div>
```

Not sure all this templating is actually necessary, we just want to display some non-cropped text.